tests/gha-run-k8s-common.sh

```diff
@@ -104,10 +104,14 @@ function get_cluster_credentials() {
 
 function delete_cluster() {
     test_type="${1:-k8s}"
-
-    az group delete \
-        -g "$(_print_rg_name ${test_type})" \
-        --yes
+    local rg
+    rg="$(_print_rg_name ${test_type})"
+
+    if [ "$(az group exists -n "${rg}")" == "true" ]; then
+        az group delete \
+            -g "${rg}" \
+            --yes
```

Comment:
```suggestion
        az group delete -g "${rg}" --yes
```
Tiny bit more readable.